src/ts/app/console/src/reducer.ts

```diff
@@ -10,7 +10,7 @@ export interface ReducerInputs {
 }
 
 export function createReducer(inputs: ReducerInputs): ImprovedLoopReducer<Subtree, HandledAction, any> {
-    return function reduce(previous: Subtree, action: HandledAction): Subtree | [Subtree, HandledAction] {
+    return function reduce(previous: Subtree, action: HandledAction): Subtree | [Subtree, any] {
```

Comment:
why? does't this break the typing?

Reply:
There's no `DispatchedAction` yet (which is the type that's _returned_ from the reducer), so it needs to be `any`.